bson/bson.go

```diff
@@ -130,10 +130,10 @@ type Setter interface {
 	SetBSON(raw Raw) error
 }
 
-// SetZero may be returned from a SetBSON method to have the value set to
+// ErrSetZero may be returned from a SetBSON method to have the value set to
 // its respective zero value. When used in pointer values, this will set the
 // field to nil rather than to the pre-allocated value.
-var SetZero = errors.New("set to zero")
+var ErrSetZero = errors.New("set to zero")
```

Comment:
Potentially breaks backwards compat for little gain I'd say. @domodwyer, not sure how likely is this to be used?

Reply:
I don't think this is much of an issue, it's unlikely to be used by 99% of people.

Reply:
FYI, I am one of the 1% that's affected by this change. Add reactions here for "me too".